3.test_cases/21.torchtitan/slurm/1.llama_3_8b_torchtitan.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+#SBATCH --nodes=1 # number of nodes to use
+#SBATCH --job-name=torchtitan #name of your job
+#SBATCH --output=logs/%x_%j.out # logfile for stdout
+#SBATCH --exclusive # job has exclusive use of the resource, no sharing
+
+set -ex;
+
+###########################
+###### User Variables #####
+###########################
+
+GPUS_PER_NODE=8
+
+###########################
+## Environment Variables ##
+###########################
+
+export NCCL_DEBUG=INFO
+export FI_PROVIDER=efa
+export FI_EFA_USE_HUGE_PAGE=0 
+## Switching SYNC_MEMOPS to zero can boost throughput with FSDP
+## Disables CU_POINTER_ATTRIBUTE_SYNC_MEMOPS
+## Reduces memory synchronizations
+## https://docs.nvidia.com/cuda/cuda-driver-api/group__CUDA__UNIFIED.html
+export FI_EFA_SET_CUDA_SYNC_MEMOPS=0
+# LD_PRELOAD is required for PyTorch to find the NCCL library
+# This path assumes you are using the Deep Learning AMI
+# If you are not using the DLAMI, you may need to update this path
+export LD_PRELOAD=/usr/local/cuda-12.1/lib/libnccl.so
+export NCCL_SOCKET_IFNAME=^docker,lo,veth_def_agent,eth
```

Comment:
```suggestion
export NCCL_SOCKET_IFNAME=^docker,lo,veth
```
cf. https://github.com/aws-samples/awsome-distributed-training/pull/571/files